management/client_test.go

```diff
@@ -46,6 +52,7 @@ func TestClient_Update(t *testing.T) {
 	expectedClient.ClientID = nil                       // Read-Only: Additional properties not allowed.
 	expectedClient.SigningKeys = nil                    // Read-Only: Additional properties not allowed.
 	expectedClient.JWTConfiguration.SecretEncoded = nil // Read-Only: Additional properties not allowed.
+	expectedClient.ClientSecret = nil
```

Comment:
This is so the secret doesn't get recorded in the request made to update a client. Similar logic for other occurrences below.